src/pybind/mgr/dashboard/frontend/angular.json

```diff
@@ -56,6 +56,7 @@
               }
             ],
             "styles": [
+              "node_modules/swagger-ui/dist/swagger-ui.css",
```

Comment:
Not sure how ok would this be (CC @alfonsomthd ) but perhaps removing this import here and importing this from the component .scss file itself or even the `styleUrls` in the `@Component` decorator? That way we wouldn't add this stylesheets at global scope...

Reply:
It does not work due to Angular encapsulation.

Reply:
Following [this](https://github.com/angular/angular/issues/25160) , it seems there's no alternative out yet for 
ng-deep . Here's some [WIP](https://drafts.csswg.org/css-scoping-1/) for the alternative.